src/home-task-3/routes/index.ts

```diff
@@ -96,4 +108,68 @@ router.get(
   }),
 );
 
+router.post(
+  '/api/add-user-to-group',
+  routeHandler<Request>(async (req, res) => {
+    const payload: UserGrouptInput = req.body;
+    const userToGroup = await addToGroup(payload);
```

Comment:
for boolean variables, it is better to start with "is..." or "has...". in this case it would be great if you have named this as `const isUserAddedToGroup = ...`

naming should always make sense especially in big projects with more than one developer